components/button/button.tsx

```diff
@@ -160,7 +160,8 @@ export default class Button extends React.Component<ButtonProps, any> {
     const iconType = loading ? 'loading' : icon;
     const iconNode = iconType ? <Icon type={iconType} /> : null;
     const needInserted = React.Children.count(children) === 1 && (!iconType || iconType === 'loading');
-    const kids = children ? React.Children.map(children, child => insertSpace(child, needInserted)) : null;
+    // tslint:disable-next-line:max-line-length
+    const kids = children || children === 0 ? React.Children.map(children, child => insertSpace(child, needInserted)) : null;
```

Comment:
hmm... if children is `false`, what is expected to display?

Reply:
an empty button